doc/dev/kubernetes.rst

```diff
@@ -166,40 +171,33 @@ build, and the Ceph build too.
 5. Run a Rook cluster
 =====================
 
-.. note::
-
-    This is just some basic instructions: the Rook documentation
-    is much more expansive, at https://github.com/rook/rook/tree/master/Documentation
+Please refer to `Rook's documentation <https://github.com/rook/rook/tree/master/Documentation>`_
+for setting up a Rook operator, a Ceph cluster and the toolbox.
 
 The Rook source tree includes example .yaml files in
-``cluster/examples/kubernetes/ceph/``.  The important ones are:
-
-- ``operator.yaml`` -- runs the Rook operator, which will execute any other
-  rook objects we create.
-- ``cluster.yaml`` -- defines a Ceph cluster
-- ``toolbox.yaml`` -- runs the toolbox container, which contains the Ceph
-  CLI client.
-
-Copy these into a working directory, and edit as necessary to configure
+``cluster/examples/kubernetes/ceph/``. Copy these into
+a working directory, and edit as necessary to configure
 the setup you want:
 
-- Ensure that the ``image`` field in the operator matches the built Ceph image
+- Ensure that the ``image`` field in the ``operator.yaml`` matches the built Ceph image
   you have uploaded to your Docker repository.
-- Edit the ``storage`` section of the cluster: set ``useAllNodes`` and
-  ``useAllDevices`` to false if you want to create OSDs explicitly
-  using ceph-mgr.
-    
+- Ensure that ``spec.cephVersion.image`` points to your docker registry::
+
+    spec:
+      cephVersion:
+        allowUnsupported: true
+        image: 192.168.122.1:5000/ceph/ceph:latest
+
 Then, load the configuration into the kubernetes API using ``kubectl``:
 
 ::
 
-    kubectl apply -f ./operator.yaml 
-    kubectl apply -f ./cluster.yaml 
-    kubectl apply -f ./toolbox.yaml 
+    kubectl apply -f ./cluster.yaml
```

Comment:
The quickstart guide uses the example `cluster-test.yaml`, which will only create a single osd per node in the simplest configuration possible. Since this is a testing scenario, how about if we use cluster-test.yaml here as well.